mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionsProvider.scala

```diff
@@ -330,3 +336,6 @@ class CompletionsProvider(
     end match
   end completionItems
 end CompletionsProvider
+
+case object Scala3CursorString:
+  val value = "CURSOR"
```

Comment:
cursor came here

Reply:
Maybe we could put it into Completions companion object under `CURSOR` value?

Reply:
It won't be a big improvement, would it? Since it is used only in `CompletionProvider` and `AmmoniteFileCompletion`

Reply:
I don't think we need the additional `Scala3CursorString` object. 

Alternatively, you could rename it to `Cursor`  Scala3 part is redundant since we are in Scala 3 sources and String is not really true since it's an object

Reply:
done

Reply:
Not a strong opinion, but I'm wondering why we wrap the cursor by `case object Cursor` instead of just defining the constant value `CURSOR`? I don't think there would have other members under `Cursor` case object other than `value`.

Reply:
hmm, currently it is accessed by both `CompletionProvider` and `AmmoniteFileCompletion`, so if defined as a constant, it had to be passed all along the way as an argument from `CompletionProvider` through `Completions` to reach `AmmoniteFileCompletion` as a value. Or, it could be wrapped in an object to be statically available where it is needed.